runner.py

```diff
@@ -1,12 +1,19 @@
-import os
-from application.image_caption_generator import ImageCaptionGenerator
+import utils
+from application.image_caption_generator import start
+
+
+"""
+To run the terminal application, make sure your terminal is in the root directory of the project. 
+In that directory, run 'python -m runner'. This runs this file as a module of the entire project, allowing imports to 
+function properly.
+"""
+
+
+def main():
+    utils.clear()
+    print('Welcome to the Ovative Group Caption Generator!\n')
+    start()
+
 
 if __name__ == '__main__':
-    os.system('cls' if os.name == 'nt' else 'clear')
-    print('Welcome to the Ovative Group Image Caption Generator!', end='\n\n')
-    image_caption_generator = ImageCaptionGenerator()
-    image_caption_generator.start()
-
-# To run the test suite, make sure your terminal is in the root directory of the project
-# In the terminal, run 'python -m runner'
-# This runs this file as a module of the entire project, allowing imports to function properly
+    main()
```

Comment:
More of a nitpick, but maybe im wrong: If this is all we're going to put into the main method, why make one? We can just have the if statement and then the utils command, print line, and start() after it and get rid of the main call. Open for discussion though, just an out there thought.

Reply:
I was looking at good Python practices, and this was recommended for a file like this. I think we should keep it like this for the convention. If this turns out not to be good practice, we can remove it.